getting-started/about.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
This seems to be a duplicate of `pages/about.md`

Reply:
Noted.